easybuild/easyconfigs/r/R/R-3.2.3-intel-2016a-libX11-1.6.3.eb

```diff
@@ -42,14 +42,14 @@ configopts += " --with-recommended-packages=no"
 # since we rely on the updated easyblock for R to configure correctly w.r.t. BLAS/LAPACK
 easybuild_version = '3.5.0'
 
-name_tmpl = '%(name)s_%(version)s.tar.gz'
+local_name_tmpl = '%(name)s_%(version)s.tar.gz'
 exts_default_options = {
     'source_urls': [
         'http://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
         'http://cran.r-project.org/src/contrib/',  # current version of packages
         'http://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
     ],
-    'source_tmpl': name_tmpl,
+    'source_tmpl': local_name_tmpl,
```

Comment:
Just inline local_name_tmpl in these 2016a and 2016b configs like with the latter R configs?
```python
'source_tmpl': '%(name)s_%(version)s.tar.gz',
```

Reply:
@Micket ok, fixed in 500095395e